Cannot build from master !! ??

```console
$ ./hack/build.sh 
+ LAUNCH_PATH=/root/go/src/github.com/openshift/installer
++ dirname ./hack/build.sh
+ cd ./hack/..
++ go list -e -f '{{.Dir}}' github.com/openshift/installer
+ PACKAGE_PATH=/root/go/src/github.com/openshift/installer
+ test -z /root/go/src/github.com/openshift/installer
+ LOCAL_PATH=/root/go/src/github.com/openshift/installer
+ test /root/go/src/github.com/openshift/installer '!=' /root/go/src/github.com/openshift/installer
+ MODE=release
++ git describe --always --abbrev=40 --dirty
+ LDFLAGS=' -X main.version=v0.3.0-145-g6106d58ddf61f3f18c51f6b295ab8c73160dac91'
+ TAGS=
+ OUTPUT=bin/openshift-install
+ export CGO_ENABLED=0
+ CGO_ENABLED=0
+ case "${MODE}" in
+ TAGS=' release'
+ test '' '!=' y
+ go generate ./data
writing assets_vfsdata.go
+ echo ' release'
+ grep -q libvirt_destroy
+ go build -ldflags ' -X main.version=v0.3.0-145-g6106d58ddf61f3f18c51f6b295ab8c73160dac91' -tags ' release' -o bin/openshift-install ./cmd/openshift-install
# github.com/openshift/installer/vendor/k8s.io/client-go/transport
vendor/k8s.io/client-go/transport/round_trippers.go:437:9: undefined: strings.Builder
```

--- a/hack/build.sh
+++ b/hack/build.sh
@@ -2,7 +2,7 @@
 
 set -ex
 
-version() { echo "$@" | gawk -F. '{ printf("%03d%03d%03d\n", $1,$2,$3); }'; }
+version() { IFS="."; printf "%03d%03d%03d\\n" $@; }
 
 average_go_version=1.10
 current_go_version=$(go version | cut -d " " -f 3)

--- a/hack/build.sh
+++ b/hack/build.sh
@@ -2,6 +2,16 @@
 
 set -ex
 
+version() { echo "$@" | gawk -F. '{ printf("%03d%03d%03d\n", $1,$2,$3); }'; }
+
+average_go_version=1.10
+current_go_version=$(go version | cut -d " " -f 3)
+
+if [ "$(version "${current_go_version#go}")" -lt "$(version "$average_go_version")" ]; then
+     echo "Go version should be greater or equal to $average_go_version"
+     exit 1
+fi
+
 LAUNCH_PATH="${PWD}"
 cd "$(dirname "$0")/.."
 

--- a/hack/build.sh
+++ b/hack/build.sh
@@ -2,13 +2,14 @@
 
 set -ex
 
-version() { IFS="."; printf "%03d%03d%03d\\n" $@; }
+# shellcheck disable=SC2068
+version() { IFS="."; printf "%03d%03d%03d\\n" $@; unset IFS;}
 
-average_go_version=1.10
+minimum_go_version=1.10
 current_go_version=$(go version | cut -d " " -f 3)
 
-if [ "$(version "${current_go_version#go}")" -lt "$(version "$average_go_version")" ]; then
-     echo "Go version should be greater or equal to $average_go_version"
+if [ "$(version "${current_go_version#go}")" -lt "$(version "$minimum_go_version")" ]; then
+     echo "Go version should be greater or equal to $minimum_go_version"
      exit 1
 fi
 

